test/integration/CloseImmatureLongs.t.sol

```diff
@@ -178,4 +176,23 @@ contract PricingTest is EverlongTest {
         );
         vm.stopPrank();
     }
+
+    /// @dev Tests the situation where the closing of an immature position
+    ///      results in losses that exceed the amount of assets owed to the
+    ///      redeemer who forced the position closure.
+    function test_immature_losses_exceed_assets_owed() external {
+        // Deploy Everlong.
+        deployEverlong();
+
+        // Make a large deposit.
+        depositEverlong(10_000e18, bob);
+
+        // Ensure previewRedeem returns zero for a small amount of shares.
+        uint256 assetsOwed = everlong.previewRedeem(1_000);
+        assertEq(assetsOwed, 0);
+
+        // Ensure revert when attempting to redeem a small amount of shares.
+        vm.expectRevert(IEverlong.RedemptionZeroOutput.selector);
+        redeemEverlong(1_000, bob);
+    }
```

Comment:
It would be better to make this a fuzz test with clearly defined ranges where we expect this to be true.

Reply:
Agreed. Have updated this accordingly. Couldn't think of a good way to derive the value for `_shareAmount` since it's incredibly small, but let me know if you do :)